lambdas/tests/unit/services/session-request-validator.test.ts

```diff
@@ -122,11 +117,125 @@ describe("session-request-validator.ts", () => {
+
+            jest.spyOn(jwtVerifier.prototype, "verify").mockReturnValue(Promise.resolve(jwtPayload));
+
+            const payload = (await sessionRequestValidator.validateJwt(
+                Buffer.from("test-jwt"),
+                client_id,
+            )) as JWTPayload;
+
+            await expect(payload).toEqual(jwtPayload);
+        });
+
+        it("should fail to validate the jwt if scope is not openid", async () => {
+            const client_id = "request-client-id";
+            const redirect_uri = "redirect-uri";
+            const scope = "test";
+            const state = "state";
+
+            jest.spyOn(jwtVerifier.prototype, "verify").mockReturnValue(
+                Promise.resolve({
+                    client_id: client_id,
+                    redirect_uri: redirect_uri,
+                    scope: scope,
+                    state: state,
+                    shared_claims: personIdentity,
+                } as JWTPayload),
+            );
+
+            await expect(async () =>
+                sessionRequestValidator.validateJwt(Buffer.from("test-jwt"), client_id),
+            ).rejects.toThrow(
+                expect.objectContaining({
+                    message: "Session Validation Exception",
+                    details: "Invalid scope parameter",
+                }),
+            );
+        });
+
+        it("should fail to validate the jwt if scope is missing", async () => {
+            const client_id = "request-client-id";
+            const redirect_uri = "redirect-uri";
+            const state = "state";
+
+            jest.spyOn(jwtVerifier.prototype, "verify").mockReturnValue(
+                Promise.resolve({
+                    client_id: client_id,
+                    redirect_uri: redirect_uri,
+                    state: state,
+                    shared_claims: personIdentity,
+                } as JWTPayload),
+            );
+
+            await expect(async () =>
+                sessionRequestValidator.validateJwt(Buffer.from("test-jwt"), client_id),
+            ).rejects.toThrow(
+                expect.objectContaining({
+                    message: "Session Validation Exception",
+                    details: "Invalid scope parameter",
+                }),
+            );
+        });
+
+        it("should fail to validate the jwt if state is missing", async () => {
+            const client_id = "request-client-id";
+            const redirect_uri = "redirect-uri";
+            const scope = "openid";
+
+            jest.spyOn(jwtVerifier.prototype, "verify").mockReturnValue(
+                Promise.resolve({
+                    client_id: client_id,
+                    redirect_uri: redirect_uri,
+                    scope: scope,
+                    shared_claims: personIdentity,
+                } as JWTPayload),
+            );
+
+            await expect(async () =>
+                sessionRequestValidator.validateJwt(Buffer.from("test-jwt"), client_id),
```

Comment:
This is the only variable required all the other should have been inline the same goes for the other. But this all adequate and it doesn't have to be perfect.